src/lightning_app/cli/lightning_cli_remove.py

```diff
@@ -0,0 +1,17 @@
+import click
+
+from lightning_app.cli.cmd_ssh_keys import SSHKeyManager
+
+
+@click.group("remove")
+def cli_remove() -> None:
+    """Remove Lightning AI self-managed resources (ssh-keys, etc…)"""
+    pass
+
+
+@cli_remove.command("ssh-key")
+@click.argument("key_id")
+def remove_ssh_key(key_id: str) -> None:
+    """Remove a Lightning AI ssh-key from your account."""
```

Comment:
Was it meant to say this?
```suggestion
    """Remove a ssh-key from your Lightning AI account."""
```